vmdb/product/toolbars/container_service_center_tb.yaml

```diff
@@ -16,9 +16,11 @@
       :text: 'Edit this #{ui_lookup(:table=>"container_service")}'
       :title: 'Edit this #{ui_lookup(:table=>"container_service")}'
       :url: '/edit'
+      :enabled: 'false'
     - :button: container_service_delete
       :image: delete
       :text: 'Remove this #{ui_lookup(:table=>"container_service")} from the VMDB'
       :title: 'Remove this #{ui_lookup(:table=>"container_service")} from the VMDB'
       :url_parms: '&refresh=y'
       :confirm: 'Warning: This #{ui_lookup(:table=>"container_service")} and ALL of its components will be permanently removed from the Virtual Management Database.  Are you sure you want to remove this #{ui_lookup(:table=>"container_service")}?'
+      :enabled: 'false'
```

Comment:
@fryguy @erinboyd no we don't add enabled: false in toolbars for buttons on summary screen, we turn them on/off in application_helper method when screen loads. @erinboyd  since you don't need these buttons at this time i would say remove them from toolbar and add them back when you need them. Also remove the corresponding features from product features for these buttons.